ddtrace/tracer.py

```diff
@@ -143,10 +143,27 @@ def __init__(
                 sync_mode=self._use_sync_mode(),
             )
         self.writer = writer  # type: TraceWriter
-        self._partial_flush_enabled = asbool(get_env("tracer", "partial_flush_enabled", default=False))
-        self._partial_flush_min_spans = int(
+
+        # TRACER_PARTIAL_FLUSH_ENABLED should be deprecated after version 1.0.0 is released
```

Comment:
we need to do this in a way where we log this for customers. we want to be sure if someone is using it, they get a notice to change to the new one. a code comment won't be too helpful